lib/public/views/Runs/ActiveColumns/runDetectorsActiveColumns.js

```diff
@@ -10,20 +10,58 @@
  * granted to it by virtue of its status as an Intergovernmental Organization
  * or submit itself to any jurisdiction.
  */
+import { h } from '/js/src/index.js';
+import { frontLink } from '../../../components/common/navigation/frontLink.js';
 import { formatDetectorQuality } from '../format/formatDetectorQuality.js';
 
 /**
  * Factory for detectors related active columns configuration
- * @param {Detector[]} detectors detectors list
+ * @param {DplDetector[]} dplDetectors detectors list
```

Comment:
You should be more careful with breaking changes!!!!

Check the usages of this function, and you'll see what is wrong...
If this function was not supposed to be reused anywhere else, it should have been named differently

Reply:
Not enough. I insist, you really need to be more careful on this to avoid introducing regression!

Reply:
And as I said, if this is run detectors active columns specifically for QC/RCT/Whatever, it should have a more specific name (if you do so, think about a very clear name, and think about the existing usages).

Reply:
splitted for Sync and Async ActiveColumns